main.tex

```diff
@@ -770,16 +773,19 @@ \subsubsection*{Provenance entities}
 Ideally, it can be linked to a complete experiment specification (e.g., as a SED-ML~\cite{Koehn2008} or SESSL~\cite{Ewald2014} file or simply as the execution code in a general purpose programming language) and to documentation in a standard format that applies reporting guidelines such as MIASE for simulation experiments~\cite{Waltemath2011}.
 Diverse simulation experiments might be applied to serve the analysis, calibration, and validation of a simulation model.
 
-To further structure the set of simulation experiments applied, we distinguish simulation experiments whether they are used for optimization, sensitivity analysis, perturbation, parameter scan, steady-state analysis, or time course analysis.
-This list is neither complete nor are the categories disjoint, and, given a different set of simulation studies, they will likely be subject to renaming, extension, and refinement.
-We have identified optimization experiments to be all experiments where an implicit or explicit objective function is being used.
+To further structure the set of applied simulation experiments, we distinguish simulation experiments by whether they are used for optimization, sensitivity analysis, perturbation, parameter scan, steady-state analysis, or time course analysis.
```

Comment:
Moved "applied" to before "simulation experiments" and added "by" before "whether". The diff isn't working here

Reply:
oh the diff is working now, awesome!

Reply:
haha, not for me ;)